src/nyx/featureset.h

```diff
@@ -457,6 +457,15 @@ namespace Nyxus
 		WEIGHTED_HU_M6,
 		WEIGHTED_HU_M7,
 
+		// Image Quality features
+		FOCUS_SCORE,

```

Comment:
I suggest naming these and future image quality features in a distinctive way, for example, IMQ_FOCUS_SCORE, IMQ_SHARPNESS, etc in order to emphasize their applicability to the whole slide, not to a ROI.